summit/2018.html

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+permalink: /summit/
+category: summit
+---
+
+<div class="summit-2018">
+<!-- Intro Header -->
+<header class="intro">
+    <div class="intro-body-summit">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-10 col-md-offset-1">
+                    <h1 class="brand-heading">
+                        <img src="/images/summit/logo.svg"
+                            alt="Open Source Design Summit 2018"/>
+                    </h1>
+                    <h1 class="headline">November 2-4<span>th</span> 2018
+                    </h1>
+                    <h2 class="location">Tirana, Albania
+                    </h2>
+                    <p class="tagline">Bringing designers and developers together to<br />improve Design and Usability in Open Source</p>
+                    <a href="#" class="button" target="_blank">Register</a>
```

Comment:
This links don't go anywhere. Same for Support or Code of Conduct. 
Register should redirect towards Open Collective Event, Support should direct toward Open Collective, and for Code of Conduct not sure what's the plan.

Reply:
We can leave the CoC for later, I guess; I have added the 2 other links.